tests/integration-tests/tests/efa/test_efa.py

```diff
@@ -44,11 +39,8 @@ def test_efa(
 
     Grouped all tests in a single function so that cluster can be reused for all of them.
     """
-    # We collected OSU benchmarks results for c5n.18xlarge only.
-    osu_benchmarks_instances = ["c5n.18xlarge"]
 
-    # 32 instances are required to see performance differences in collective OSU benchmarks.
-    max_queue_size = 32 if instance in osu_benchmarks_instances else 2
+    max_queue_size = 2
```

Comment:
Minor: since this is now constant we can put it in the cluster config and remove the variable injection.